apps/sms/index.html

```diff
@@ -71,18 +71,18 @@
     <script defer src="shared/js/fb/fb_data_reader.js"></script>
     <script defer src="shared/js/fb/fb_reader_utils.js"></script>
     <script defer src="shared/js/event_dispatcher.js"></script>
-    <script defer src="js/navigation.js"></script>
-    <script defer src="js/inter_instance_event_dispatcher.js"></script>
-    <script defer src="js/time_headers.js"></script>
-    <script defer src="js/contacts.js"></script>
-    <script defer src="js/drafts.js"></script>
-    <script defer src="js/threads.js"></script>
-    <script defer src="js/utils.js"></script>
+    <script defer src="views/shared/js/navigation.js"></script>
+    <script defer src="views/shared/js/inter_instance_event_dispatcher.js"></script>
+    <script defer src="views/shared/js/time_headers.js"></script>
+    <script defer src="views/shared/js/contacts.js"></script>
```

Comment:
unsure for contacts -- it's mostly a service, and also we'll likely want to handle this differently when we'll have the DB... but mozContacts is not available in workers. We'll see later, let's put it there for now